### ISSUE TITLE
Ordered list bullet list showing boxes in front on copying pastin from ms-outlook and ms-word

### ISSUE THREAD

**Issue by muditgoel2016** [tbluemel/rtf.js#62]
On copying-pasting ordered/bullet list from outlook/ms-word the rtf file does not render properly. 

- The first item of the list in the clipboard (the item from where the copying action began) renders with a  bunch of boxes in front.

-The bullet list icons don't look quit right.
See the image below:
![image](https://user-images.githubusercontent.com/18715605/83430216-e08f9080-a452-11ea-83b1-930d7ad2c8d8.png)
Please, find rtf file inside the zip that will replicate the issue
[list_improper_render.zip](https://github.com/tbluemel/rtf.js/files/4712351/list_improper_render.zip)

**Comment by muditgoel2016** [comment]
@zoehneto Hi, your previous fix worked as expected. Could you or request someone else to take up a look at this bug as well as it is blocking a complete acceptance of rtfjs base rendering in my application. TIA

**Comment by zoehneto** [comment]
I had a quick look at the issue but I don't see an immediate fix for this and I won't have time to look at it within the next two weeks. PRs are of course always welcome, otherwise you can filter the HTML for the two or three invalid characters.

**Comment by muditgoel2016** [comment]
> I had a quick look at the issue but I don't see an immediate fix for this and I won't have time to look at it within the next two weeks. PRs are of course always welcome, otherwise you can filter the HTML for the two or three invalid characters.

Yup, thanks! Whenever you get time; until then, I will filter out the invalid chars.
